js/main.js

```diff
@@ -0,0 +1,25 @@
+const $photoURL = document.querySelector('#photo-url');
+
+$photoURL.addEventListener('input', function (event) {
+  const $urlValue = $photoURL.value;
+  const $imgSrc = document.querySelector('.url-pic');
```

Comment:
You should do this `querySelector` globally rather than inside this function. The event is going to fire every single time a user presses a key, so we'd have to traverse the DOM over and over each time to find this element. If we just did it once globally, we wouldn't need to traverse the DOM over and over